package.json

```diff
@@ -15,6 +15,7 @@
     "express": "^4.17.1",
     "fast-csv": "^4.3.6",
     "jsonwebtoken": "^8.5.1",
+    "jwt-decode": "^3.1.2",
```

Comment:
No need additional library, `jwt.verify` already provides the decode. Can remove